scripts/installscripts/stretch-install-default.sh

```diff
@@ -387,7 +387,7 @@ sudo iwconfig wlan0 power off
 
 # Install required packages
 sudo apt-get update
-sudo apt-get --yes --force-yes install apt-transport-https samba samba-common-bin python-dev python-pip gcc linux-headers-4.9 lighttpd php7.0-common php7.0-cgi php7.0 php7.0-fpm at mpd mpc mpg123 git ffmpeg python-mutagen python3-gpiozero
+sudo apt-get --yes --force-yes install apt-transport-https samba samba-common-bin python-dev python-pip gcc linux-headers-4.9 lighttpd php7.0-common php7.0-cgi php7.0 php7.0-fpm php7.0-mbstring php7.0-opcache at mpd mpc mpg123 git ffmpeg python-mutagen python3-gpiozero
```

Comment:
fpm was removed in #839, don't introduce here again?

Reply:
nah, that would not be a wise move :-D
That's why there are conflicts with the develop branch.